packages/stream-tests/src/__tests__/model-instance-document.test.ts

```diff
@@ -514,16 +504,32 @@ describeIfV3ShouldPass('ModelInstanceDocument API multi-node tests', () => {
   let midMetadata: ModelInstanceDocumentMetadataArgs
 
   beforeAll(async () => {
-    ipfs0 = await createIPFS()
-    ipfs1 = await createIPFS()
+    ipfs0 = await createIPFS({
+      rust: {
+        type: 'binary',
+        network: Networks.INMEMORY,
+      },
+    })
+    ipfs1 = await createIPFS({
+      rust: {
+        type: 'binary',
+        network: Networks.INMEMORY,
+      },
+    })
     await swarmConnect(ipfs0, ipfs1)
 
-    ceramic0 = await createCeramic(ipfs0)
-    ceramic1 = await createCeramic(ipfs1)
+    ceramic0 = await createCeramic(ipfs0, { networkName: Networks.INMEMORY })
```

Comment:
ditto, can we make this the default behavior of `createCeramic`?

Reply:
Ready